site/docs/getting-started/design.mdx

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
I was tempted to suggest removing this for the same reasons as the component pages. However, let's keep the 3 pages under getting started as it could be useful for our design & content colleagues to see how things behave when we have a docs section like this.